careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn what it's like to work at the CFPB
+                    </a>
+                </p>
+            </section>
+        </div>
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Follow us on LinkedIn</h2>
+                <p class="short-desc">
+                    The CFPB is one of the most searched for agencies
+                    in the federal government.
+                    Connect with us to stay updated on the work
+                    we do and new opportunities to be a part of it.
+                </p>
+                <p class="short-desc">
+                    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+                       class="jump-link jump-link__right">
+                        Go to our LinkedIn
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Provide Us With Feedback</h2>
+                <p class="short-desc">
+                    {# TODO: Have Content look at this paragraph
+                    and the weird link break. #}
+
+                    We take pride in our feedback-centered culture
+                    that helps us recruit great people.
+                    If you have any feedback, or any specific questions,
+                    let us know at:
+                </p>
+                <p class="short-desc">
+                    <ul class="list list__icons">
+                        <li class="list_item">
+                            <span class="cf-icon cf-icon-email list_icon"></span>
+                            <span class="h5 list_text">Email:</span>
+                            <p><a href="mailto:jobs@consumerfinance.gov"
+                               class="jump-link jump-link__right">
+                                jobs@consumerfinance.gov
+                            </a></p>
+                        </li>
+                    </ul>
+                </p>
+            </section>
+        </div>
+    </section>
+
+    {# Footer #}
+    <aside>
+        <div class="block u-mb0 u-mt0 block__flush-sides block__bg">
+            <div class="">
+                    <!-- TODO: Add Office of Civil Rights url -->
+                    {%- import "related-links.html" as related_links -%}
+                    {{- related_links.render([
+                        [ '/blog/',
+                          'CFPB Blog' ],
+                        [ '/newsroom/',
+                          'Newsroom' ],
+                        [ '/equal-employment-opportunity/',
+                          'Equal Employment Opportunity' ]
```

Comment:
There was a discrepancy between the mockup and the finalized copy. Can
@sarahsimpson08 weigh in one too?
On Jun 26, 2015 7:23 AM, "Ans" <notifications@github.com> wrote:

> In careers/students-and-graduates/index.html
> <https://github.com/cfpb/cfgov-refresh/pull/680#discussion_r33350771>:
>
> > +        </div>
> > +    </section>
> > +
> > +    {# Footer #}
> > +    <aside>
> > +        <div class="block u-mb0 u-mt0 block__flush-sides block__bg">
> > +            <div class="">
> > +                    <!-- TODO: Add Office of Civil Rights url -->
> > +                    {%- import "related-links.html" as related_links -%}
> > +                    {{- related_links.render([
> > +                        [ '/blog/',
> > +                          'CFPB Blog' ],
> > +                        [ '/newsroom/',
> > +                          'Newsroom' ],
> > +                        [ '/equal-employment-opportunity/',
> > +                          'Equal Employment Opportunity' ]
>
> This should be Office of Civil Rights.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/cfgov-refresh/pull/680/files#r33350771>.
>